### DISCUSSION TITLE
How to properly uninstall/remove?

### DISCUSSION THREAD

**Discussion by Authcursed** [TT-ReBORN/Georgia-ReBORN#189] [Q&A]
I want to uninstall/remove georgia-reborn because unfortunately it lags and crash on my poor laptop :(
How do i uninstall the right way? I'm using the standard foobar, so I installed on the appdata folder not on the program files folder

**Comment by TT-ReBORN** [comment]
Hi @Authcursed,

I do not know which version you have installed, but you can try the latest DEV version which will be soon RC-3 from this link here: https://github.com/TT-ReBORN/Georgia-ReBORN/archive/refs/heads/master.zip

If you have an old laptop with slow CPU, you can set the performance to "Low quality" or "Lowest quality" in top `Menu` > `Settings` > `Theme performance`.

---

### For standard installation removal:

To remove ONLY Georgia-ReBORN, just delete the `georgia-reborn` folder from `C:\Users\YourUsername\AppData\Roaming\foobar2000`.

If you want complete removal with its components and related stuff, you need to additionally delete these folders: `biography`, `configuration`, `georgia-reborn`, `library` and `user-components`.

The cleanest removal would be to uninstall foobar from Windows > Control panel > Programs and Features and uninstall foobar2000.

-TT
